### ISSUE TITLE
Unable to log in to Google drive through Cryptomator on iOS

### ISSUE THREAD

**Issue by ataraxia28** [cryptomator/ios#182]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/cryptomator/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/cryptomator/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

I am unable to log in to my Google drive account through Cryptomator on iOS. 

### What software is involved?

- Operating System: iOS 15.3.1
- Cryptomator: 2.1.2

### Steps to Reproduce

1. Select add vault on Cryptomator 2 and subsequently select open existing vault on Google Drive
2. tap add account
3. enter google account login credentials and attempt to log in

### Expected Behavior

I thought I would be able to log in to google drive to access my stored vault on the cloud.

### Actual Behavior

After attempting to log in to google drive through cryptomator 2 a window appears stating that cookies are disabled despite having them enabled on my mobile browser settings. 

### Reproducibility

Always

### Anything else?

![image 1](https://user-images.githubusercontent.com/100528455/155891602-0419fb29-8a7e-4f60-a2a0-e4c6a4caa1d2.jpg)
![image 2](https://user-images.githubusercontent.com/100528455/155891605-d632521b-6e2e-4eee-bca9-b7da3f641a97.png)

**Comment by tobihagemann** [comment]
You've already mentioned that cookies are enabled in Safari but have you tried to clear the cookies? https://support.apple.com/en-us/HT201265

Do you have any Safari plugins/extensions installed like 1Blocker? If yes, you might have to change its settings.

**Comment by ataraxia28** [comment]
> You've already mentioned that cookies are enabled in Safari but have you tried to clear the cookies? https://support.apple.com/en-us/HT201265
> 
> Do you have any Safari plugins/extensions installed like 1Blocker? If yes, you might have to change its settings.

Yes, I have tried clearing the cookies and website data but I still encountered the same problem. I also do not have any extensions or blockers installed on Safari as well.


However, I just managed to resolve the issue by deleting the same vault that I initially imported through the _Files_ app on my iPhone as a workaround. Perhaps that was the issue on my end and it was not allowing me to have two of the same vaults active that were derived from the same master key.
